deploy-manage/security/secure-settings.md

```diff
@@ -16,267 +15,208 @@ mapped_pages:
+* **Changes require a restart**: Most secure settings take effect only after restarting the nodes. However, some are marked as *reloadable* and can be updated without a restart using the [`reload_secure_settings`](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-nodes-reload-secure-settings) API. Refer to [Reloadable settings](#reloadable-secure-settings) for more information.
+* **Keystore is per-node**: Each node in the cluster has its own keystore file. Secure settings must be specified individually on every node and must have the same values across the cluster. Orchestrated deployments, such as ECH, ECE, and ECK, handle this automatically when configuring secure settings.
 
-With the {{es}} keystore, you can add a key and its secret value, then use the key in place of the secret value when you configure your sensitive settings.
+::::{tip}
+To check whether a setting can be added to the keystore and supports reloading without a restart, look for the `Secure` and `Reloadable` qualifiers in the [{{es}} configuration reference](elasticsearch://reference/elasticsearch/configuration-reference/index.md).
+
+For example, you can search for `secure_bind_password` in the [security settings document](elasticsearch://reference/elasticsearch/configuration-reference/security-settings.md) to see how these qualifiers appear in context.
+::::
+
+The instructions below cover how to manage {{es}} keystore settings for each deployment type.
 
 :::::{tab-set}
 :group: deployment-type
 
-::::{tab-item} {{ecloud}}
+::::{tab-item} ECH and ECE
 :sync: cloud
+% ### ECE and ECH
+You can manage {{es}} secure settings in the **Security > {{es}} keystore** section of your deployment page in the Elastic Cloud Console or ECE Cloud UI.
 
-There are three types of secrets that you can use:
+:::{note}
+{{ece}} also supports managing {{es}} keystore of your deployments through its [RESTful API](https://www.elastic.co/docs/api/doc/cloud-enterprise/). Refer to [Configure {{es}} keystore through ECE API](cloud://reference/cloud-enterprise/ece-restful-api-examples-configuring-keystore.md) for an example.
+:::
 
-* **Single string** - Associate a secret value to a setting.
-* **Multiple strings** - Associate multiple keys to multiple secret values.
-* **JSON block/file** - Associate multiple keys to multiple secret values in JSON format.
+There are three input formats you can use for secure setting values:
 
+* **Single string**: Associate a secret value to a setting.
+* **Multiple strings**: A group of key-value pairs, each stored as part of the setting's value.
+* **JSON block/file**: A structured JSON object containing multiple key-value pairs, typically used for more complex secrets like service account credentials.
 
-**Add secret values**
+### Add secure settings [ec-add-secret-values]
```

Comment:
I see your point about the sidebar. try these as H4s and I think they'll be excluded from OTP